src/lab12.py

```diff
@@ -406,7 +421,7 @@ def __init__(self):
         self.url = None
         self.scroll = 0
 
-        self.measure_raster_and_draw = MeasureTime("raster-and-draw")
+        self.measure = MeasureTime("browser.trace")
```

Comment:
Put this behind a flag, so it doesn't constantly create files unless we wan them.

Reply:
You mean in our impl, not in the book, right? Can do, no problem.